server/commands/check.go

```diff
@@ -44,6 +44,14 @@ func NewCheckQuery(datastore storage.OpenFGADatastore, t trace.Tracer, m metric.
 
 // Execute the query in `checkRequest`, returning the response or an error.
 func (query *CheckQuery) Execute(ctx context.Context, req *openfgapb.CheckRequest) (*openfgapb.CheckResponse, error) {
+	//if rand.Intn(100) < 10 { // 10% of failure
```

Comment:
Mind doing a quick pass to removed commented out code and debug print statements? I've seen a couple in this PR.